cloudsync/provider.py

```diff
@@ -50,6 +51,7 @@ class Provider(ABC):                    # pylint: disable=too-many-public-method
     sep: str = '/'                            ; """Path delimiter"""
     alt_sep: str = '\\'                       ; """Alternate path delimiter"""
     oid_is_path: bool = False                 ; """Objects stored in cloud are only referenced by path"""
+    events_have_path: bool = False            ; """Item path is included in events. Applicable for OID providers only"""
```

Comment:
i really dont like any of these top level config items.... can we simply assume that events have a path if they have a path, and that they cannot be filtered if they don't.

Reply:
agree, derived behavior would be much cleaner

might be able to remove this new attr in favor of event.path (more detail in thread below)